velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -260,6 +260,68 @@ struct FromUnixtimeFunction {
   bool isConstantTimeFormat_{false};
 };
 
+template <typename T>
+struct ToUTCTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void initialize(
+      const core::QueryConfig& config,
+      const arg_type<Varchar>* /*input*/,
+      const arg_type<Varchar>* timezone) {
+    if (timezone != nullptr) {
```

Comment:
nit: maybe use `if（timezone)` to simplify.

Reply:
Done, thanks!